index.qmd

```diff
@@ -38,9 +38,9 @@ Regardless, users will access data over a network, which must be considered when
 
 As a community, we have arrived at the following **cloud-optimized format pattern:**
 
-1. Metadata includes addresses for data blocks.
+1. Metadata or specification provides addresses for data blocks.
```

Comment:
This revision generalizes the pattern to also support Zarr, where the data block addresses are denoted by the standard itself rather than the metadata